tutorials/cloud-functions-github-container-builder/index.md

```diff
@@ -262,4 +262,4 @@ You can check the [logs][logs] for errors.
 
 Section [Set up automated builds](#set-up-automated-builds) is derived from
 @ahmetb's [Automated Static Website Publishing with Cloud Container
-Builder](../automated-publishing-container-builder/index.md) tutorial.
+Builder](https://cloud.google.com/community/tutorials/automated-publishing-cloud-build) tutorial.
```

Comment:
I think you're supposed to omit the domain and start with `/community` directly.

See https://sourcegraph.com/search?q=repo:%5Egithub%5C.com/GoogleCloudPlatform/community%24++%5C%28/community/tutorials

Reply:
Thanks. 

Relative links like that work for the rendered result on cloud.google.com, but I don't think that they'll work when someone is viewing the tutorial directly in GitHub.

Reply:
Sounds like we need to establish a guideline around this as clearly there are tutorials doing both styles.

Reply:
Agreed. That's now on my to-do list.